src/Blorc.OpenIdConnect/Extensions/JsonElementExtensions.cs

```diff
@@ -0,0 +1,68 @@
+namespace Blorc.OpenIdConnect
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Security.Claims;
+    using System.Text.Json;
+
+    public static partial class JsonElementExtensions
+    {
+      public static IEnumerable<Claim> AsClaims(this JsonElement element, string claimType = "") {
```

Comment:
intendation seems off here? Please use format (in VS it's CTRL K + D) to format the file.